src/components/global/Column.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Droppable } from "react-beautiful-dnd";
+import "../../stylesheets/components/global/column.scss";
```

Comment:
Do not import styles like this, we've been putting all the imports in index.scss. See the current-task component for an example.